README.md

```diff
@@ -140,6 +140,8 @@ jobs:
         uses: platisd/duplicate-code-detection-tool@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          # Leave the report as one comment
+          one_comment: true
```

Comment:
I'd say move these lines to the end so they are not confused with `directories` which is a compulsory argument.

Also perhaps turn into:
```suggestion
          # Leave only one comment with the report and update it for consecutive runs
          one_comment: true
```

Reply:
And one more question about bool(only_code). Such a transformation can give true, if only_code is not an empty string.
Need to use one_code in ("true", "1").